[Bug Report] CONTRIBUTING.md guide needs work

### Describe the bug

I tried following the CONTRIBUTING.md guide to run `pytest` but it does not work.

I first did the following, which worked.

```bash
sudo apt update
sudo apt install swig

pip install -e .[all]
```

Then I tried running `pytest` but I received the following error message:

```
/workspaces/PettingZoo $ pytest -v
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: -n=auto
  inifile: /workspaces/PettingZoo/pyproject.toml
  rootdir: /workspaces/PettingZoo
```

I also tried using `make test-all`:

```
/workspaces/PettingZoo $ make test-all
pytest -v --cov=pettingzoo --cov-report term
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: -n=auto --cov=pettingzoo --cov-report
  inifile: /workspaces/PettingZoo/pyproject.toml
  rootdir: /workspaces/PettingZoo

make: *** [Makefile:11: test-all] Error 4
```

### Code example

_No response_

### System info

I installed PettingZoo in the default GitHub codespace configuration.

### Additional context

_No response_

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/PettingZoo/issues) in the repo

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -22,7 +22,7 @@ Contributing code is done through standard github methods:
 4. Submit a pull request. It will be reviewed by maintainers and they'll give feedback or make requests as applicable
 
 ### Considerations
-- Make sure existing tests pass (`pip install -e .[all]` and then run `pytest -v` -- may also need to `apt-get`/`brew` `install swig` and `AutoROM -v`)
+- Make sure existing tests pass (`pip install -e .[testing,all]` and then run `pytest -v` -- may also need to `apt-get`/`brew` `install swig` and `AutoROM -v`)
 - Make sure your new code is properly tested and fully-covered
 - Any fixes to environments should include fixes to the appropriate documentation
 - Changes to environment functionality should be avoided when reasonable, and when they occur the environment version must be bumped.

